package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "did-driver",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.1.0",
```
Universal resolver refers to this driver as version 0.1 [here](http://github.com/evannetwork/universal-resolver/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R67). Or should universal resolvers reference be updated?